Sources/Common/RxManager.swift

```diff
@@ -71,6 +71,43 @@ extension Reactive where Base: RxManager {
             }
         }
     }
+    /**
+     Returns a `Single` of (OperaResponse?,T) for the current request. Notice that T conforms to OperaDecodable. If something goes wrong an `OperaSwift.Error` error is propagated through the result sequence.
+
+     - parameter route: the route indicates the networking call that will be performed by including all the needed information like parameters, URL and HTTP method.
+     - parameter keyPath: keyPath to look up json object to serialize. Ignore parameter or pass nil when json object is the json root item.
+
+     - returns: An instance of `Single<(OperaResponse?,T)>`
+     */
+    func objectResponse<T: OperaDecodable>(_ route: RouteType, keyPath: String? = nil) -> Single<(OperaResponse?, T)> {
+        return base.rx.response(route).flatMap { operaResult -> Single<(OperaResponse?, T)> in
+            let serialized: DataResponse<T>  = operaResult.serializeObject(keyPath)
+            switch serialized.result {
+            case .failure(let error):
+                return Single.error(error)
+            case .success(let anyObject):
+                return Single.just((operaResult.operaResponse, anyObject))
```

Comment:
what about:

```
guard let response = operaResult.operaResponse else {
     return Single.error(CUSTOM_ERROR)
}
return Single.just((response, anyObject))
```
This way we could get rid of the optional `OperaResponse`.

Also, what do you think about naming that tuple? So the final user can do `$0.response` and `$0.object` instead of `$0.0` and `$0.1`. I would define a `typealias` as `(response: OperaResponse, object: T)` and replace each tuple with this type. What do you think?